### DISCUSSION TITLE
Showing data from array with objects

### DISCUSSION THREAD

**Discussion by sindregusdal** [Budibase/budibase#10722] [Help]
Hi. I use Firestore as my data source, and I read data from a collection called "orders". 

Within each order (document in firestore) we have a field called "cart". This field is an array with typical cart data (itemname, qty, options, price aso)

Is it possible in Budibase, to create a table, where clicking a ROW opens a new screen/modal to show all cart data?

**Comment by melohagan** [comment]
Hey @sindregusdal 

You should be able to accomplish this, but you may need a second query that can get cart data for a given filter.

There's more on filtering in the docs: https://docs.budibase.com/docs/firestore#using-filter-fields

Then you can pass binding values through to your custom queries as shown here: https://docs.budibase.com/docs/data-sources#custom-queries
